mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -1869,6 +1869,27 @@ func TestAccContainerCluster_withSoleTenantGroup(t *testing.T) {
 	})
 }
 
+func TestAccContainerCluster_withAdvancedMachineFeatures(t *testing.T) {
+	t.Parallel()
+
+	resourceName := fmt.Sprintf("tf-test-%s", randString(t, 10))
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerCluster_withAdvancedMachineFeatures(resourceName),
```

Comment:
Do you mind adding a second test step that updates the `threads_per_core` to a new value, or removes it? I want to make sure we exercise updating it during a test .

Reply:
Fixed!